tardis_kromer_plot.py

```diff
@@ -304,16 +304,105 @@ def line_in_L(self):
     @property
     def line_info(self):
         """produces list of elements to be included in the kromer plot"""
-        line_out_infos_within_xlims = self.line_out_infos.loc[
-            (self.line_out_infos.wavelength >= self._xlim[0])
-            & (self.line_out_infos.wavelength <= self._xlim[1])
+
+        # gets list of elements and number of emitted packets
+        self.last_line_interaction_out_id = self.line_out_infos
+        self.last_line_interaction_out_angstrom = self.line_out_nu.to(
+            units.Angstrom, equivalencies=units.spectral()
+        )
+
+        self.lines = self.mdl.lines
+
+        packet_out_filter = (
```

Comment:
packet_out_filter doesn't seem to be used anywhere in the code. Is this filter necessary?

Reply:
see comment above